### ISSUE TITLE
Doesn't annotate module().decorator

### ISSUE THREAD

**Issue by mattlewis92** [olov/ng-annotate#204]
Angular 1.4 introduced a new decorator helper with the pattern:
```
angular
  .module('moduleName', [])
  .decorator('serviceName', function($delegate, and, other, injectables) {
  });
```

However it's not currently automatically annotated.

**Comment by yggie** [comment]
having the same problem, I had to remove the `decorator` from the chain otherwise `ng-annotate` ignores the module entirely:

```javascript
var app = angular.module('moduleName', []);

app.decorator('serviceName', [/* service names */ function(/* services */) {
});
```

**Comment by fracz** [comment]
+1

**Comment by olov** [comment]
See more info for why this has been the case in #82. I'll change the code to support `foo.decorator("yo", function() ...)` generically with an exception for `$stateProvider.decorator(...)` instead. Will be out in next release.

**Comment by olov** [comment]
released in 1.1.0
